nearuplib/util.py

```diff
@@ -9,17 +10,65 @@
 from nearuplib.constants import S3_BUCKET
 
 
+class NetworkError(Exception):
```

Comment:
can you move this to a seperate file: nearuplib/exceptions.py

Reply:
done!